src/app/analytics/analytics.factory.js

```diff
@@ -12,9 +12,10 @@ function analyticsFactory($window, $timeout, sessionService) {
         var item, donationType;
 
         // Instantiate cart data layer
+        const hash = sha3(cartData.id, { outputLength: 20*4 });
```

Comment:
It might be better to hard code `80` and add a comment as to why we need this to output a string with length of 20.